app/data/chips.json

```diff
@@ -9,7 +19,7 @@
         "Mempool-policy"
       ],
       "activated": "No"
-    },
+    },    
```

Comment:
I see a trailing tab here 😅 But not really a big deal. Will remove later (also this file is automatically processed by Forestry, so it might remove it)

Reply:
Oh sorry.

I created this via the Github web interface.

Will take care not to repeat this in future PRs.